images/inventory-generator/root/usr/local/bin/generate

```diff
@@ -0,0 +1,348 @@
+#!/bin/env python
```

Comment:
Note: the "traditional" path/shebang is `#!/usr/bin/env python`, though this should also work on Fedora/RHEL/CentOS.